src/conventional/metar_csv2ioda.py

```diff
@@ -0,0 +1,261 @@
+#!/usr/bin/env python
+
+#
+# (C) Copyright 2020 UCAR
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# author: Greg Thompson gthompsn AT ucar DOT edu
+# This script will work with decoded, CSV files created by gthompsn stored at NCAR
+#   data-access.ucar.edu:/glade/campaign/ral/aap/gthompsn/METARs/2019/20191231/2019123118_metars.csv.gz
+#
+
+from __future__ import print_function
+import sys
+import math
+from datetime import datetime
+from argparse import ArgumentParser, ArgumentDefaultsHelpFormatter
+from pathlib import Path
+from csv import DictReader
+import netCDF4
+
+IODA_CONV_PATH = Path(__file__).parent/"@SCRIPT_LIB_PATH@"
```

Comment:
This file is intended to be processed through CMake. Change the extension to .py.in to reflect this.

Reply:
It isn't clear to me about adding the ".in" extension - there's an entire directory of 'marine' that do not have the py.in file names.  I was following others' examples.  What's the choice here - I can easily do either, but was trying to be consistent with marine subdir.

Reply:
We used to end the files in .in but somewhere along the way it was changed to just be .py. See for example in the gsi-ncdiag directory. This happened in PR 263 https://github.com/JCSDA-internal/ioda-converters/pull/263 by @markjolah